osidb/models.py

```diff
@@ -381,15 +387,19 @@ class FlawHistoryResolution(models.TextChoices):
 
     # , from srtnotes "mitigate"
     mitigated_by = models.CharField(
-        choices=FlawMitigate.choices, max_length=10, blank=True
+        choices=FlawMitigate.choices, max_length=10, blank=True, default=""
```

Comment:
This will fail the validation, since we do not have `""` as an option in the  `FlawMitigate.choices`.

Reply:
That is true. Then I wonder why don't we have a `null=True` there

Reply:
Because of this https://docs.djangoproject.com/en/4.1/ref/models/fields/#django.db.models.Field.null

That was one of the main points of reviewing the required fields. Basically you don't want to have `null=True` on string based fields (Char/Text) since then you have the two possible representations of the empty values (empty string vs. NULL/None)

Reply:
Then the default is needed as the value is obviously not mandatory so providing none should not result in a failure. Which means that the problem is not here but at the model. And as I look at it I am sure that missing empty value is not the only issue there. The real issue is that `mitigated_by` is multiple choice field and not just char. This might be probably wrong from the beginning.

Reply:
Actually we are commenting the model and not the migration :smile: so yeah it is wrong but the default is not the main issue here.

Reply:
I actually recalled this https://issues.redhat.com/browse/OSIDB-126 so I will remove the field instead.

Reply:
@Elkasitu @JakubFrejlach this will make the next OSIDB release basically a breaking API change as `mitigated_by` will disappear from the API...

Reply:
@osoukup please make sure that the removal is N-1 compatible, even if the field is of no use we don't know who is using it and how, and then we should make a post to osidb-announce mentioning that it's being deprecated / removed

Reply:
or just keep the field and we will remove it during the next major version release

Reply:
there is probably a way to mark it as deprecated in the OpenAPI schema

Reply:
Oh, now I remember, this is the problematic multiplevalued field. Good that we are free to remove it.

This seems like the way how to mark [field deprecated](https://drf-spectacular.readthedocs.io/en/latest/drf_spectacular.html#drf_spectacular.utils.extend_schema_serializer)